app/views/participants/_summary.html.haml

```diff
@@ -0,0 +1,11 @@
+%li.participant-summary.participant-summary--with-thumbnail
+  = link_to participant_url(participant), class: 'participant-summary__url' do
+    - if participant.photo.present?
```

Comment:
When the participant has no photo, the alignment of style is a bit dirty.
![2016-07-28 12 15 45](https://cloud.githubusercontent.com/assets/290782/17200099/43b0f530-54bd-11e6-9b08-97544c4a6d1a.png)